toolkit/smart-contracts/offchain/src/init_governance/mod.rs

```diff
@@ -57,11 +58,16 @@ where
 }
 
 #[derive(serde::Serialize)]
+/// Result type of [run_init_governance]. Serialized to JSON.
```

Comment:
There is nothing about JSON in this type.
```suggestion
/// Result type of [run_init_governance].
```